tools/gen_nmake.mk

```diff
@@ -14,7 +14,7 @@ Makefile.nmake tst.nmake: FORCE
 	@echo ''			>> $@
 	@echo '# This file can be auto-regenerated with $$make -f Makefile.unx Makefile.nmake' >> $@
 	@echo ''			>> $@
-	@echo -n 'objs =' >> $@
+	@/bin/echo -n 'objs =' >> $@
```

Comment:
Why is it needed for only these two lines? because of the -n parameter?

Reply:
yes.

Ii seems like I can use `printf` alternatively, though.

Reply:
Could be a better choice, knowing that this is hardcoding the path.

Reply:
@pablodelara done.